src/queries/$query.ts

```diff
@@ -1,7 +1,7 @@
-import type { QueryOptions, TableWatcherCallbacks, TableWatcherParams } from "@/queries/types";
+import type { QueryOptions, TableWatcherParams, WatcherOptions } from "@/queries/types";
+import type { BaseTables, Tables } from "@/tables/types";
```

Comment:
Seeing an error here

Reply:
Oh yes, I didn't push some part of my wip for the next PR. Fixed, sorry